src/core/utils.ts

```diff
@@ -0,0 +1,137 @@
+import { init as Cuid2 } from '@paralleldrive/cuid2';
+import MersenneTwister from './MersenneTwister';
+import type { Core } from './core';
+
+const LOREM =
+	'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
+const PARAGRAPHS = [LOREM];
+const SENTENCES = LOREM.replace(/\. /g, '.\n').split('\n');
+const WORDS = LOREM.toLowerCase().replace(/[,.]/, '').split(' ');
+
+const CHAR_CODE_0 = 48;
+const CHAR_CODE_LOWERCASE_Z = 122;
+
+export class Utils {
```

Comment:
What about `Randomization` (or something similar) instead of `Util`?

Reply:
Sooo... for the most part, yes, the methods are used for randomizing, with the exception of 2.

`filterChecks` is a strongly typed union discriminator for zod def checks and `noop` is just a single no operation reference.

`noop` only gets used in 2 places (currently) but `filterChecks` is used a LOT.

Should we move randomization to its own class and variable and keep `filterChecks` in utils?

Reply:
Hmmm I kinda like that's all in one place, but on the other hand...
Let's think on this for a while, and if we can't come up with something better we can release this as is.